scenario_configs/eval6/carla_mot/carla_mot_adversarialpatch_undefended_coco.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
perhaps going a step further, is there a need for having the `coco_format` parameter to toggle between in the first place? As opposed to just having things always be in coco format. This would simplify things a bit

Reply:
That is true. I kept them in because the `coco_format` flag goes so many different places it's easy to mess up.

Reply:
There is so much hardcoded MOT format stuff, that it's probably useful to have ways of toggling that. What about having `"coco_format": true` for all the configs and removing the non-coco format ones? Then, if someone really wants to use MOT format, they know exactly which things to switch.

Reply:
yea i'm good with that

Reply:
Done